### DISCUSSION TITLE
When will the javascript edition come out?

### DISCUSSION THREAD

**Discussion by ChrisTritton** [smartcontractkit/full-blockchain-solidity-course-py#8] [Q&A] [answered]
Hi, very happy that this comprehensive tutorial now exists in the world. Should I wait for the JS version though? I don't work with python at all in real life and would like to get used to JS in blockchain. Thanks!

**Comment by CJ-Rose** [comment, marked as answer]
Agreed! This is the only beginner's resource I have found that isn't 2+ years old. Just FYI I'm 2 hours in and nothing language-specific yet. We are starting with basics of Solidity, which is most similar to Javascript. My personal opinion ... if you're paying attention to what's going on in the space, there's just no time to waste. I need to get moving with these skills ASAP so I can start participating. Good luck!

**Reply by PatrickAlphaC** [reply]
Spot on!!

**Comment by PatrickAlphaC** [comment]
Great question all! 

The first 5 hours or so is almost all solidity, and will give you a lot of important foundational knowledge. Being able to switch back and forth between languages will be a _huge_ benefit for you anyways. 

And quite personally, I think your life will be a _lot_ easier doing smart contract development with python > javascript.

Good luck!
